lib/index.js

```diff
@@ -55,6 +56,16 @@ function setupRateLimitEventExt(server, options) {
   const extractKey = options.extractKey;
   const onError = options.onError;
 
+  const limitResponseHandler = options.limitResponseHandler || ((result, req, reply) => {
```

Comment:
Minor:
We could move the default handler at the root of the file, out of the `setupRateLimitEventExt` function (it is not closing on any other variable).

Reply:
in this case it is mostly the same because this is a builder not used at high rate, I'd prefer to keep the function as closed as possible to their usage unless there are perf reasons not to do it.